eslint.config.js

```diff
@@ -34,6 +34,32 @@ export default tsEslint.config(
       "@typescript-eslint/consistent-type-exports": "error",
       "@typescript-eslint/no-import-type-side-effects": "error",
       "@typescript-eslint/consistent-type-imports": "error",
+      "@typescript-eslint/use-unknown-in-catch-callback-variable": "off",
```

Comment:
Things I think are true:
- errors should be `unknown`
- your tsconfig settings should be strict

Things I want to be true:
- that ts setting works in `then().catch()` (it does not yet)
- [I don't have to fix it myself](https://typescript-eslint.io/rules/use-unknown-in-catch-callback-variable/#when-not-to-use-it) (apparently you can override the global, but i don't want to do that.

Therefore, I don't think this rule is important enough (and i don't feel like making that many code changes to support it 😬 )

Reply:
I can't say I've encountered this but your answer seems reasonable to me.

Reply:
TS made the unknown thing a default a bit ago, and I got wrecked with a refactor of tons of tiny changes from `any` to `unknown`. But apparently, that was only for try/catch, not then().catch()

So we wait on...